homeassistant/components/google_assistant/trait.py

```diff
@@ -1050,6 +1051,8 @@ def query_attributes(self):
             # open, even if that is currently incorrect.
             if self.state.attributes.get(ATTR_ASSUMED_STATE):
                 response['openPercent'] = 50
+            elif self.state.state == STATE_UNKNOWN:
```

Comment:
See my comment with the response from Google. We should raise `NotSupported` error instead when it's an assumed state or unknown state.